spec/workers/articles/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe Articles::BustCacheWorker, type: :worker do
```

Comment:
for the sake of incorporating the newly merged changes could you take a look at the additions here? I did remove the article object because we don't need it: https://github.com/thepracticaldev/dev.to/pull/5301/files#diff-0d3fc1fbde4c04d90da9619f0992bd8b

the structure of the test (grouping all the lets in the `with article` section) is intented to shorten the distance between the setup objects and the tests that use them

Reply:
I can definitely rearrange this spec like the new one. 

I purposefully removed the `enqueues_jobs` specs bc I think that is basically testing the Sidekiq implementation. Sidekiq should enqueue the job and [that logic is tested within the Sidekiq gem](https://github.com/mperham/sidekiq/blob/33aedf62dee55a08474792ce9e34ca5f061874d3/test/test_worker.rb#L24) so there is no need for us to test it as well.

Reply:
Agreed! I think the main goal of that was to check the name of the queue the job was in more than anything, so we might want to add a test to make sure `BustCacheWorker` is on the right queue

Reply:
Agreed! I think the main goal of that was to check the name of the queue the job was in more than anything, so we might want to add a test to make sure `BustCacheWorker` is on the right queue